Geometry/MuonCommonData/python/testGEMXML_cfi.py

```diff
@@ -0,0 +1,117 @@
+import FWCore.ParameterSet.Config as cms
+
+# This config was generated automatically using generate2023Geometry.py
+# If you notice a mistake, please update the generating script, not just this config
+
+XMLIdealGeometryESSource = cms.ESSource("XMLIdealGeometryESSource",
+    geomXMLFiles = cms.vstring(
+        'Geometry/CMSCommonData/data/materials.xml',
```

Comment:
Hi @ianna @bsunanda - I presume this hierarchy needs to be updated to be compatible with 91x?